### PR TITLE
Introduce module legend-extension-format-graphql

### PR DESCRIPTION
**Pull request by abhishoya-gs**
## Summary
This change introduces a new module legend-extension-query-graphql for the purpose of supporting GraphQL stereotypes Query, Mutation, and Subscription. The same stereotypes in legend-engine are present [here](https://github.com/finos/legend-engine/blob/master/legend-engine-xt-graphQL-pure/src/main/resources/core_external_query_graphql/binding/profile.pure).

## How did you test this change?

- [ ] Test(s) added
- [x] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

GraphQLStereotypes

### CODE REVIEW

File: packages/legend-extension-query-graphql/package.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@finos/legend-extension-query-graphql",
```
Comment (akphi) [inline packages/legend-extension-query-graphql/package.json:2]: let's do `@finos/legend-extension-format-graphql` instead

File: packages/legend-extension-query-graphql/package.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    "npm-run-all": "4.1.5",
+    "rimraf": "4.1.2",
+    "sass": "1.58.3",
+    "typescript": "4.9.5"
+  },
+  "publishConfig": {
+    "directory": "build/publishContent"
+  },
+  "extensions": {
+    "graphManagerPreset": "@finos/legend-graph-manager-preset-query-graphql",
```
Comment (akphi) [inline packages/legend-extension-query-graphql/package.json:59]: let's update these accordingly

File: packages/legend-extension-query-graphql/src/Query_GraphQL_Extension.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import packageJson from '../package.json';
+import { AbstractPreset } from '@finos/legend-shared';
+import { Query_GraphQL_PureGraphManagerPlugin as Query_GraphQL_PureGraphManagerPlugin } from './graphManager/Query_GraphQL_PureGraphManagerPlugin.js';
+import { Query_GraphQL_PureProtocolProcessorPlugin as Query_GraphQL_PureProtocolProcessorPlugin } from './graphManager/protocol/pure/Query_GraphQL_PureProtocolProcessorPlugin.js';
+
+export class Query_GraphQL_GraphManagerPreset extends AbstractPreset {
```
Comment (akphi) [inline packages/legend-extension-query-graphql/src/Query_GraphQL_Extension.ts:22]: we should rename these files to `FMT_GraphQL...`

**Review by akphi — CHANGES_REQUESTED** [review]

File: packages/legend-extension-query-graphql/package.json (hunk 1 above)
Comment (abhishoya-gs) [reply packages/legend-extension-query-graphql/package.json]: done

File: packages/legend-extension-query-graphql/package.json (hunk 2 above)
Comment (abhishoya-gs) [reply packages/legend-extension-query-graphql/package.json]: done

File: packages/legend-extension-query-graphql/src/Query_GraphQL_Extension.ts (hunk 3 above)
Comment (abhishoya-gs) [reply packages/legend-extension-query-graphql/src/Query_GraphQL_Extension.ts]: done

File: packages/legend-extension-format-graphql/src/FMT_GraphQL_Extension.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import packageJson from '../package.json';
+import { AbstractPreset } from '@finos/legend-shared';
+import { Query_GraphQL_PureGraphManagerPlugin as Query_GraphQL_PureGraphManagerPlugin } from './graphManager/FMT_GraphQL_PureGraphManagerPlugin.js';
+import { Query_GraphQL_PureProtocolProcessorPlugin as Query_GraphQL_PureProtocolProcessorPlugin } from './graphManager/protocol/pure/FMT_GraphQL_PureProtocolProcessorPlugin.js';
+
+export class Query_GraphQL_GraphManagerPreset extends AbstractPreset {
```
Comment (akphi) [inline packages/legend-extension-format-graphql/src/FMT_GraphQL_Extension.ts:22]: don't forge to update the name of these classes to use the right prefix `FMT_`

File: packages/legend-extension-format-graphql/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# @finos/legend-extension-format-graphql
```
Comment (akphi) [inline packages/legend-extension-format-graphql/CHANGELOG.md:1]: let's delete this file, this package is never released, it should not have a changelog file

File: packages/legend-extension-format-graphql/package.json [hunk 6]
```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@finos/legend-extension-format-graphql",
+  "version": "1.0.0",
+  "description": "Legend extension for GraphQL Query format",
```
Comment (akphi) [inline packages/legend-extension-format-graphql/package.json:4]:
```suggestion
  "description": "Legend extension for GraphQL",
```

File: packages/legend-extension-format-graphql/package.json [hunk 7]
```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@finos/legend-extension-format-graphql",
+  "version": "1.0.0",
```
Comment (akphi) [inline packages/legend-extension-format-graphql/package.json:3]: Changeset will automatically bump this upon release, this should be set to `0.0.0` for now

```suggestion
  "version": "0.0.0",
```

File: packages/legend-extension-format-graphql/package.json [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+    "npm-run-all": "4.1.5",
+    "rimraf": "4.1.2",
+    "sass": "1.58.3",
+    "typescript": "4.9.5"
+  },
+  "publishConfig": {
+    "directory": "build/publishContent"
+  },
+  "extensions": {
+    "graphManagerPreset": "@finos/legend-graph-manager-preset-query-graphql",
```
Comment (akphi) [inline packages/legend-extension-format-graphql/package.json:59]:
```suggestion
    "graphManagerPreset": "@finos/legend-graph-manager-preset-format-graphql",
```

File: packages/legend-extension-format-graphql/package.json [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+    "pureProtocolProcessorPlugin": "@finos/legend-pure-protocol-processor-plugin-query-graphql",
```
Comment (akphi) [inline packages/legend-extension-format-graphql/package.json:60]:
```suggestion
    "pureProtocolProcessorPlugin": "@finos/legend-pure-protocol-processor-plugin-format-graphql",
```

File: packages/legend-extension-format-graphql/package.json [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+    "pureGraphManagerPlugin": "@finos/legend-pure-graph-manager-plugin-query-graphql"
```
Comment (akphi) [inline packages/legend-extension-format-graphql/package.json:61]:
```suggestion
    "pureGraphManagerPlugin": "@finos/legend-pure-graph-manager-plugin-format-graphql"
```

**Review by akphi — CHANGES_REQUESTED** [review]

File: packages/legend-extension-format-graphql/src/FMT_GraphQL_Extension.ts (hunk 4 above)
Comment (abhishoya-gs) [reply packages/legend-extension-format-graphql/src/FMT_GraphQL_Extension.ts]: thanks, updated

File: packages/legend-extension-format-graphql/CHANGELOG.md (hunk 5 above)
Comment (abhishoya-gs) [reply packages/legend-extension-format-graphql/CHANGELOG.md]: done

File: packages/legend-extension-format-graphql/package.json (hunk 7 above)
Comment (abhishoya-gs) [reply packages/legend-extension-format-graphql/package.json]: done

File: packages/legend-extension-format-graphql/package.json (hunk 8 above)
Comment (abhishoya-gs) [reply packages/legend-extension-format-graphql/package.json]: done

File: packages/legend-extension-format-graphql/package.json (hunk 10 above)
Comment (abhishoya-gs) [reply packages/legend-extension-format-graphql/package.json]: done

**Review by akphi — APPROVED** [review]

**Pull request merged**